### DISCUSSION TITLE
ppstructure中文命名pdf转word

### DISCUSSION THREAD

**Discussion by running-az** [PaddlePaddle/PaddleOCR#14125] [Q&A]
执行：python3 predict_system.py \
    --image_dir=ppstructure/docs/recovery/测试.pdf \
    --recovery=True \
    --use_pdf2docx_api=True \
    --output=../output/
    生成的中间文件 xxxx(乱码)show_1.jpg 。。。。
    请问是不支持中文命名的pdf文件吗？

**Comment by UserWangZz** [comment]
试试用英文名的文件，可能是python读取文件的问题
